src/test/run-pass/inc-range-pat.rs

```diff
@@ -1,9 +1,9 @@
 // Test old and new syntax for inclusive range patterns.
```

Comment:
See this line ^--- 😃. We want `#![allow(...)]` here.

Reply:
🤦